tests/fsharp/tests.fs

```diff
@@ -1074,9 +1074,14 @@ module CoreTests =
     let ``printing-default-stdout-47 --langversion:4_7`` () =
          printing "--langversion:4.7" "z.output.test.default.stdout.47.txt" "z.output.test.default.stdout.47.bsl" "z.output.test.default.stderr.txt" "z.output.test.default.stderr.bsl"
 
+    // For this, check both with and without refemit
     [<Test>]
-    let ``printing-default-stdout-50 --langversion:5_0`` () =
-         printing "--langversion:5.0" "z.output.test.default.stdout.50.txt" "z.output.test.default.stdout.50.bsl" "z.output.test.default.stderr.txt" "z.output.test.default.stderr.bsl"
+    let ``printing-default-stdout-50 --langversion:5_0 --refemit+`` () =
```

Comment:
This adds some testing for refemit on and off.  The "printing" test is fed into F# interactive fragment by fragment so is a useful place to locate any testing that is related to piece-by-piece evaluation of interactive fragments